app/scripts/Xprize-UserInterfacesRetargeting.xml

```diff
@@ -97,18 +97,14 @@
   </module>
 
   
-    <!--iFeelSuit Skin to multi-haptic application-->
+    <!--Weight retargeting-->
   <module>
-    <name>iFeelSuitSkinActuatorsTestApplication</name>
+    <name>WeightRetargetingModule</name>
+    <parameters>--from WeightRetargetingElbows.ini</parameters>
     <environment>YARP_FORWARD_LOG_ENABLE=1</environment>
-    <description>Run iFeelSuit sking application</description>
+    <description>Run weight retargeting application</description>
     <node>localhost</node>
-    <tag>iFeelSuit-SkinActuatorsTestApplication</tag>
-    <dependencies>
-      <port timeout="11.0">/iFeelSuit/WearableActuatorsCommand/input:i</port>
```

Comment:
Maybe we can keep the dependencies that are valid for this module, what do you say @RiccardoGrieco ?!

Reply:
I agree! Done in c88f246.